openghg/store/storage/_localzarrstore.py

```diff
@@ -331,6 +331,12 @@ def match_chunking(self, version: str, dataset: xr.Dataset) -> Dict[str, int]:
                 )
 
             logger.warning(msg)
+
+            # For coordinates we haven't chunked over we'll use the full size
+            for k in dataset.coords:
```

Comment:
```suggestion
            for k in dataset.dims:
```
Chunks and sizes are dicts keyed by dims. Usually our coord names mirror the dim names, but it's possible that they differ.